packages/apps/docs/package.json

```
@@ -24,10 +24,11 @@
     "lint:fmt": "prettier . --cache --check",
     "lint:pkg": "lint-package",
     "lint:src": "eslint src --ext .js,.ts,.jsx,.tsx,.mjs",
+    "prettier": "prettier",
     "start": "next start",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "test:watch": "vitest",
+    "test:watch": "vitest  --coverage",
```

```suggestion
    "test:watch": "vitest  --coverage",
```
we already have coverage separately then why we need with coverage again here

---

i like to see the coverage during watch and development